day-6/stefan-excel.py

```diff
@@ -1,47 +1,69 @@
-#!/usr/bin/env python
-
-# You'll need to pip import openpyxl
-import openpyxl
-
-excelFile = openpyxl.load_workbook('stefan-data.xlsx')
-
-sheet1 = excelFile.active
-
-for row in sheet1.iter_rows():
-    for cell in row:
-        if cell.value == 0:
-            cell.value = '<10'
-        elif cell.value == 1:
-            cell.value = '20'
-        elif cell.value == 2:
-            cell.value = '40'
-        elif cell.value == 3:
-            cell.value = '80'
-        elif cell.value == 4:
-            cell.value = '160'
-        elif cell.value == 5:
-            cell.value = '320'
-        elif cell.value == 6:
-            cell.value = '640'
-        elif cell.value == 7:
-            cell.value = '1280'
-        elif cell.value == 0.5:
-            cell.value = '10'
-        elif cell.value == 1.5:
-            cell.value = '30'
-        elif cell.value == 2.5:
-            cell.value = '60'
-        elif cell.value == 3.5:
-            cell.value = '120'
-        elif cell.value == 4.5:
-            cell.value = '240'
-        elif cell.value == 5.5:
-            cell.value = '480'
-        elif cell.value == 6.5:
-            cell.value = '960'
-        elif cell.value == 7.5:
-            cell.value = '1920'
-        elif cell.value == 8.5:
-            cell.value = '2560'
-
-excelFile.save('stefan-new-data.xlsx')
+#!/usr/bin/env python
+
+# You'll need to pip import openpyxl 
+import openpyxl
+import argparse
+
+parser = argparse.ArgumentParser( 
+    description='Script to convert HI data from lab readout to titers.',
+    usage='python3  stefan-excel.py -e [table.xlsx]')
+
+parser.add_argument(
+    '-e', action="store", default= None, required=True,
+    help='location of Excel file to parse')
+
+parser.add_argument(
+    '-s', type=int, default=20,
+    help='The starting dillution, default=20.')
```

Comment:
s/dillution/dilution/

Also, if you make your parser like this:

```python
    parser = argparse.ArgumentParser(
        formatter_class=argparse.ArgumentDefaultsHelpFormatter,
        .....
```

Then the automatic `--help` function argparse runs for you will print the default values - so no need to embed it in the help string yourself.